compiler/orchestrator_runtime/pash_init_setup.sh

```diff
@@ -221,6 +232,7 @@ else
     {
         local daemon_pid=$1
         ## Only wait for daemon if it lives (it might be dead, rip)
+
```

Comment:
This change seems irrelevant so could be removed. Random modifications in the file make it harder to understand what is meaningful change.